android/app/src/main/java/org/electroncash/electroncash3/Main.kt

```diff
@@ -493,6 +542,53 @@ class ChangePasswordDialog : AlertDialogFragment() {
 }
 
 
+class WalletExportDialog : AlertDialogFragment() {
```

Comment:
Just to reiterate, after seeing my comments on #1706 , if you'd still rather I use `ExportWalletDialog` instead, I'll switch.

Reply:
Like I said in the other PR, either way is fine as long as it's consistent.